package.json

```diff
@@ -42,7 +42,10 @@
         "lint:package": "sort-package-json --check 'package.json'",
         "lint:package:fix": "sort-package-json 'package.json'",
         "test": "mocha \"test/**/*.spec.js\"",
-        "test:debug": "mocha --inspect=9226 \"test/**/*.spec.js\""
+        "test:debug": "mocha --inspect=9226 \"test/**/*.spec.js\"",
+        "watch": "npm-run-all --parallel watch:dashboard watch:node-red",
+        "watch:dashboard": "npx nodemon --watch ./ui --ext '*' --exec npm run build:dev",
+        "watch:node-red": "cd ~/.node-red && npx nodemon --watch $(realpath ./node_modules/@flowfuse/node-red-dashboard/nodes) --ext '*' --exec node-red"
```

Comment:
Of the two, this one is the most potentially flaky, but will work if the developer has installed node-red as per https://dashboard.flowfuse.com/contributing/

Reply:
The `:dashboard` commands works well, still an annoying build step speed, but it's a lot cleaner than manually needing to trigger.

The `:node-red` one isn't currently working for me, and did spot `realpath: ./node_modules/@flowfuse/node-red-dashboard/nodes: No such file or directory` in the logs when running `npm run watch`

Reply:
Do you have `@flowfuse/node-red-dashboard` installed in your node-red npm modules? (check not flowforge)

Try: `~/.node-red && ls ./node_modules/@flowfuse/`

Reply:
You nailed it - working great.